api/tinygo/runnable/errors/errors.go

```diff
@@ -0,0 +1,28 @@
+//go:build tinygo.wasm
+
+package errors
+
+import (
+	"errors"
+)
+
+// RunErr adds a status code for use in FFI calls to return_result()
+type RunErr struct {
+	error
+	Code int
+}
+
+type HostErr error
+
+func NewError(code int, message string) RunErr {
```

Comment:
stuttering. `error.NewError`. I think this should be `errors.Run(err, code)`, and the host then can be `errors.Host(msg)`.

> The importer of a package will use the name to refer to its contents, so exported names in the package can use that fact to avoid repetition. 

from https://go.dev/doc/effective_go#package-names